src/test/java/unit/MountControllerTest.java

```diff
@@ -0,0 +1,171 @@
+package unit;
+
+import static org.mockito.Mockito.*;
+
+import bio.terra.cli.app.utils.LocalProcessLauncher;
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.businessobject.Resource;
+import bio.terra.cli.businessobject.Workspace;
+import bio.terra.cli.utils.OSFamily;
+import bio.terra.cli.utils.mount.MountController;
+import bio.terra.cli.utils.mount.MountControllerFactory;
+import bio.terra.cli.utils.mount.handlers.BaseMountHandler;
+import bio.terra.cli.utils.mount.handlers.GcsFuseMountHandler;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.stubbing.Answer;
+
+@Tag("real-unit")
+public class MountControllerTest {
+
+  @TempDir static Path tempWorkspaceDir;
+  @Mock private static MockedStatic<LocalProcessLauncher> mockStaticLocalProcessLauncher;
+
+  @BeforeAll
+  public static void setup() {
+    mockStaticLocalProcessLauncher = mockStatic(LocalProcessLauncher.class);
+  }
+
+  @AfterAll
+  public static void tearDown() {
+    mockStaticLocalProcessLauncher.close();
+  }
+
+  /** Utility method to check if a directory exists */
+  public void verifyDirectory(Path path) {
+    File file = path.toFile();
+    assert (file.exists() && file.isDirectory());
+  }
+
+  @Test
+  @DisplayName("mountController mounts buckets on linux")
+  void testMountResources() {
+    MockedStatic<Context> mockStaticContext = mockStatic(Context.class);
```

Comment:
https://www.baeldung.com/mockito-mock-static-methods#mocking-a-no-argument-static-method

Reply:
qq: Is it okay to do this instead of try-with-resource as long as I close the mock at the end of the test? My only reason is to avoid unneeded indentation though.

Reply:
It's generally better to use a try-with-resources in case the test exits before getting to the end (e.g. due to a failed assertion)

Reply:
Oh I see.